docet-core/src/main/resources/docetpdf-cover.html

```diff
@@ -0,0 +1,9 @@
+<div id="cover">
```

Comment:
is this a default file ? can users override this ? as it is a public resource on classpath mabye it would be better to put in a specific package and not on the unnamed package

Reply:
There are 4 files with the same "issue"
* docetpdf-cover.html
* codetpdf-cover-struct.css
* docetpdf-header-foorter.html
* docetpdf-page-struct.css

They are css/html pair to handle the cover page and remaining pages headers & footers.

Users *could* override them to achieve particular needs but actually isn'r really supported (ie.: can be overridden shadowing such files from other jars... not a really clean way to override them).

We can move such files out of root folder in docet.pdf.cover and docet.pdf.page folders.
We can move docetpdf.css too out of root to docet.pdf.page folder. Such file CAN be overridden from users (configuring another file from DocetDocumentResourcesAccessor)